test/v2/load/load_generator.go

```diff
@@ -146,38 +160,43 @@ func (l *LoadGenerator) readAndWriteBlob() {
 	defer l.randPool.Put(randObj) // Return to pool when done
 
 	l.submissionLimiter <- struct{}{}
```

Comment:
I'm a bit confused- if submission and the two read methods happen sequentially, what's the point of tracking the in-progress count for each separately?

Reply:
The function `readAndWriteBlob()` is called many times in parallel. The channels here are not for counting the number, they are for preventing too many parallel operations from taking place. I'm basically just using them as semaphores.